### PR TITLE
Docs cleanup

### PR DESCRIPTION
**Pull request by Lyla-Fischer**

### CODE REVIEW

**Comment by Lyla-Fischer** [comment]
@timabbott or @hackerkid This is almost a copy-paste from the Dropbox Paper, but I had to make some changes for bullets and links that didn't get onto the clipboard.

File: docs/for-mentors/how-to-mentor.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
+#How to Mentor
```
Comment (trueskawka) [inline docs/for-mentors/how-to-mentor.md]: I think this and the next heading need a space after `#` for rendering :)

File: docs/for-mentors/how-to-mentor.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+##Know your stuff:
+
+ * Make sure you’re familiar with the Zulip development environment and our 
+   [code contribution guide](https://zulip.readthedocs.io/en/latest/code-reviewing.html) 
+   from ReadTheDocs so that you can review students’ work.
+
+ * Check out the list of [common issues](common-issues.md) that come up when 
+   working with students so that you know how to respond.
+
+ * Have [relelvent zulip resources](tabs-and-useful-links.md) available so that
```
Comment (trueskawka) [inline docs/for-mentors/how-to-mentor.md]: Could you capitalize Zulip?

File: docs/for-mentors/how-to-mentor.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+   you can teach the students.
+
+
+## Be Nice and Develop Community:
+
+ * [Be encouraging](https://paper.dropbox.com/doc/aNpjVzRthI9YjsZOGNvWn), 
+   especially because younger students are still children 
+   (13-17 years old!). 
+
+ * Follow the [Code of Conduct]
```
Comment (trueskawka) [inline docs/for-mentors/how-to-mentor.md]: The space after square brackets breaks the rendering.

File: docs/for-mentors/how-to-mentor.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+    
+    * Closing duplicated PRs and asking students to learn to work with git 
+      rather than opening a new PR for every change
+
+    * Asking students to add a link to the submission in the GCI app, if the PR 
+      has been submitted but there is no task for it - it usually means the 
+      student worked on the task before claiming it, but it also happens that 
+      students just forget to resubmit a task after updating the PR
+
+    * inviting students to chat with us on Zulip if they have trouble
```
Comment (trueskawka) [inline docs/for-mentors/how-to-mentor.md]: Capitalizing the first word would be more consistent

File: docs/for-mentors/how-to-mentor.md [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+
+    * Discuss problematic tasks on mentor chat and ask for help when needed 
+      (e.g. asking one of the more experienced mentors to review a complex task 
+      you’re not sure about)
+
+    * Sometimes students get lost, especially in the beginning, and asking them 
+      to read some introductory docs on Zulip, git etc. might be helpful
+
+    * Encourage other students to help out, since they’re waiting for their 
+      tasks to be reviewed anyway, is a great way to facilitate their learning
```
Comment (trueskawka) [inline docs/for-mentors/how-to-mentor.md]: I think it should be `it's a great way`

File: docs/for-mentors/making-tasks.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+are done, tell the admins, and they will upload the task to the GCI webapp for 
+students to claim and work on.
+
+  1. Take a task idea issue from the 
+  [task creation project](https://github.com/zulip/zulip-gci/projects/1) 
+  or a make a new task from an idea that you have on your own.
+
+  2. Create or update the task in the   
+  [tasks/2017](https://github.com/zulip/zulip-gci/tree/master/tasks/2017)
+  folder of the zulip-gci repo.
```
Comment (trueskawka) [inline docs/for-mentors/making-tasks.md]: This shows on the next line, is there a double space at the end?

File: docs/for-mentors/making-tasks.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+  6. Add an upload script to the 
+  [scripts folder](https://github.com/zulip/zulip-gci/tree/master/scripts/2017) 
+  and do a dry run with a fake API key to test it.
+
+  7. Fix any issues, repeat, and merge the tasks
+
+  8. Let the GCI admins (Lyla, Tim, Vishnu) know: we’ll bulk-create the tasks
+in the GCI webapp. 
+
+  9. If there’s a directory for the task’s submissions in the [submissions repo]
```
Comment (trueskawka) [inline docs/for-mentors/making-tasks.md]: A space after square brackets breaks the rendering.

File: docs/for-mentors/making-tasks.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+(https://github.com/zulip/zulip-gci-submissions), update the link in that
+directory to point to the 2017 version of the task (see e.g. [translations/]
```
Comment (trueskawka) [inline docs/for-mentors/making-tasks.md]: A space after square brackets breaks rendering.

File: docs/for-mentors/tabs-and-useful-links.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+1. Create a separate browser profile for GCI - you can easily do it in 
+[Chrome](https://support.google.com/chrome/answer/2364824?co=GENIE.Platform%3DDesktop&hl=en) or
+[Firefox](https://support.mozilla.org/en-US/kb/profile-manager-create-and-remove-firefox-profiles)
+
+2. Add pinned tabs
+* https://chat.zulip.org
+* https://codein.withgoogle.com/
+* https://github.com/zulip/zulip-gci-submissions
+
+3. Add a tab manager
```
Comment (trueskawka) [inline docs/for-mentors/tabs-and-useful-links.md]: The paragraphs in this point need more whitespace to be properly rendered as a part of it (I think it's 3 spaces, at least that's what worked for me in other docs) :)

**Comment by Lyla-Fischer** [comment]
Thanks!

**Pull request merged**